src/aiokem/main.py

```diff
@@ -1,3 +1,92 @@
-def add(n1: int, n2: int) -> int:
-    """Add the arguments."""
-    return n1 + n2
+"""AioKem class for interacting with Kohler Energy Management System (KEM) API."""
+
+from typing import Any
+
+from aiohttp import ClientSession, ClientTimeout
+
+AUTHENTICATION_URL = "https://kohler-homeenergy.okta.com/oauth2/default/v1/token"
+CLIENT_KEY = (
+    "MG9hMXFpY3BkYWdLaXdFekYxZDg6d3Raa1FwNlY1T09vMW9"
+    "PcjhlSFJHTnFBWEY3azZJaXhtWGhINHZjcnU2TWwxSnRLUE5obXdsMEN1MGlnQkVIRg=="
+)
+API_KEY = "pgH7QzFHJx4w46fI~5Uzi4RvtTwlEXp"
+API_BASE = (
+    "https://az-amer-prod-hems-capp.jollyglacier-72224ec0.eastus.azurecontainerapps.io"
+)
+
+
+class AioKem:
+    """AioKem class for interacting with Kohler Energy Management System (KEM) API."""
+
+    def __init__(self, session: ClientSession = None) -> None:
+        """Initialize the AioKem class."""
+        self._token: str | None = None
+        self._refresh_token: str | None = None
+        self._session = (
+            session if session else ClientSession(timeout=ClientTimeout(total=10))
+        )
+
+    async def login(self, username: str, password: str) -> None:
+        """Login to the server."""
+        headers = {
+            "accept": "application/json",
+            "authorization": f"Basic {CLIENT_KEY}",
+            "content-type": "application/x-www-form-urlencoded",
+        }
```

Comment:
Its a bit faster to use CIMultiDict and the existing `hdrs` constants from aiohttp

example https://github.com/home-assistant/core/blob/4950bda40662290b218b81158199e708d5d81b89/homeassistant/components/http/headers.py#L28

Reply:
thanks